CONTRIBUTING.md

```diff
@@ -10,20 +9,25 @@ the process.
 
 ## <a name="building"></a> Building and Testing
 
-After cloning this repository (or a fork of it), you can build and run the tests for marklogic-unit-test locally 
-via the following steps:
+After cloning this repository (or a fork of it), you can build and run the tests for marklogic-unit-test
+locally. First, create a file named `gradle-local.properties` and add 
```

Comment:
We can omit this since with Docker, we know what the admin password is. One less thing for the reader to do.